adoc/admin-updates.adoc

```diff
@@ -231,6 +231,10 @@ It is not required to reload or restart `skuba-update.timer`.
 The `--annotate-only` flag makes the `skuba-update` utility only check if updates are available and annotate the node accordingly.
 When this flag is activated no updates are installed at all.
 
+When OS updates are disabled, then you will have to manage OS updates manually. In order to do so, you will have to call `skuba-update` manually on each node.
+
```

Comment:
Add : HINT: Do NOT use "zypper up/zypper patch" as this does NOT manage the kubernetes  annotations used by kured!